Makefile

```diff
@@ -74,9 +73,9 @@ check-generate:
 .PHONY: envtest
 envtest: setup-envtest
 	source <($(SETUP_ENVTEST) use -p env); \
-		TEST_CONFIG=1 go test -v -count 1 -race ./pkg/config -ginkgo.progress -ginkgo.v -ginkgo.failFast
+		TEST_CONFIG=1 go test -v -count 1 -race ./pkg/config -ginkgo.progress -ginkgo.v -ginkgo.fail-fast
```

Comment:
--ginkgo.failFast is deprecated, use --ginkgo.fail-fast instead
Learn more at:https://onsi.github.io/ginkgo/MIGRATING_TO_V2#changed-command-line-flags